Core/OfficeDevPnP.Core/Utilities/WebParts/IWebPartPostProcessor.cs

```diff
@@ -5,6 +5,6 @@ namespace OfficeDevPnP.Core.Utilities.WebParts
 {
     public interface IWebPartPostProcessor
     {
-        void Process(WebPartDefinition wpDefinition, File webPartPage);
+        void Process(WebPartDefinition wpDefinition, Web web);
```

Comment:
Hi Sergei,
I'm sorry but we cannot introduce breaking changes into PnP Core because that would potentially hurt customers using PnP Core in production. Moreover, I think you can have access to the web object through the WebId property of the File object. Or you can eventually provide overloads and make obsolete the old ones. Thus, I would kindly ask you to fix the implementation. Thanks again.